tools/build.js

```diff
@@ -2,20 +2,29 @@ var fs = require('fs');
 var path = require('path');
 var stringify = require('json-stable-stringify');
 var commandPath = path.join(__dirname, '..', 'commands.json');
+var redisCommands = require('../');
 
 var Redis = require('ioredis');
 var redis = new Redis(process.env.REDIS_URI);
 
-redis.command(function (err, res) {
+return redis.command().then(function (res) {
```

Comment:
It is not. It's only my habit of adding a return in front of promises, if not explicitly unwanted.